allennlp/data/tokenizers/word_tokenizer.py

```diff
@@ -1,19 +1,27 @@
-from typing import List
```

Comment:
There shouldn't be a `word_tokenizer.py` file.  Just a `tokenizer.py`, and one file per tokenizer.  (If they are really simple, like the LettersDigitsTokenizer, they could possibly just be put in `tokenizer.py` under the initial class definition.  But then they are hard to find, so even if they're really small, it's still probably best to put them in their own file.)